hammerdb/setup.sh

```diff
@@ -9,27 +9,17 @@ set -x
 
 coordinator_ip_address=$1
 username=$2
-hammerdb_branch=$3
 
 driverdir="${0%/*}"
 
 hammerdb_version=$(cat ~/HAMMERDB_VERSION)
 hammerdb_dir="${HOME}"/HammerDB-"${hammerdb_version}"
 
 cd "${HOME}"
-
-wget "https://github.com/TPC-Council/HammerDB/releases/download/v${hammerdb_version}/HammerDB-${hammerdb_version}-Linux.tar.gz"
-tar -zxvf HammerDB-"${hammerdb_version}"-Linux.tar.gz
-
-# here we use our fork, because it distributed tables at the beginning, which speeds up the process
-# since we can create indexes in parallel etc.
-git clone --branch "${hammerdb_branch}" https://github.com/citusdata/HammerDB.git
-mv HammerDB/src/postgresql/pgoltp.tcl "${hammerdb_dir}"/src/postgresql/pgoltp.tcl
-mv HammerDB/config/postgresql.xml "${hammerdb_dir}"/config/postgresql.xml
-
-# cd ${hammerdb_dir}/src/postgresql
-# comment out create database and user as citus cannot do that
-# sed -i 's/CreateUserDatabase $lda $db $superuser $user $password/#CreateUserDatabase $lda $db $superuser $user $password/g' pgoltp.tcl
+git clone https://github.com/citusdata/ch-benchmark.git
```

Comment:
this would mean we rely on ch-benchmark, which is probably okay and we have a single place for the citus related changes

Reply:
I could see us changing the UX a bit if we want to encourage customers to run it. Should we make a separate branch?

Reply:
separate branch on the ch-benchmark repo you mean?

Reply:
ping

Reply:
yes

Reply:
created `test-automation` branch